test/SConscript

```diff
@@ -0,0 +1,64 @@
+import os
+
+Import( 'env' )
+Import( '*' )
+
+if 'loudnessAnalyserLibStatic' in locals() and \
+    'loudnessCorrectorLibStatic' in locals() and \
+    'loudnessToolsLibStatic' in locals() and \
+    'loudnessIOLibStatic' in locals():
+
+    # Get gtest path
+    gtest_root = GetOption('gtest')
+    gtest_include = ''
+    gtest_lib = ''
+    if gtest_root:
+        gtest_include = os.path.join( gtest_root, 'include' )
+        gtest_lib = os.path.join( gtest_root, 'lib' )
+
+    # Check libavtranscoder
+    gtestEnv = env.Clone()
+
+    gtestEnv.Append(
+        CPPPATH = [
+            gtest_include,
+        ],
+        LIBPATH = [
+            gtest_lib,
+        ],
+    )
+
+    # Check gtest
+    conf = Configure(gtestEnv)
+    if conf.CheckCXXHeader('gtest/gtest.h'):
```

Comment:
__return__ statements cannot be used outside of a function in python. And I did not succeed to simplify this code using functions. Do you have any hints?